src/VisualStudio/IntegrationTest/TestUtilities/DialogHelpers.cs

```diff
@@ -111,6 +135,17 @@ private static AutomationElement FindDialogWorker(int visualStudioHWnd, string d
             return vsAutomationElement.FindFirst(TreeScope.Descendants, elementCondition);
         }
 
+        private static AutomationElement FindDialogByNameWorker(int visualStudioHWnd, string dialogName)
```

Comment:
Share code between this and `FindDialogWorker` (the only difference is `AutomationIdProperty` vs. `NameProperty`, right?), and consider renaming it to `FindDialogByIdWorker`